manim/mobject/svg/text_mobject.py

```diff
@@ -575,3 +579,334 @@ def change_alignment_for_a_line(self, alignment, line_no):
                     ]
                 )
             )
+
+
+class PangoText(SVGMobject):
+    """Display (non-LaTeX) text rendered using `Pango <https://pango.gnome.org/>`_.
+
+    PangoText objects behave like a :class:`.VGroup`-like iterable of all characters
+    in the given text. In particular, slicing is possible.
+
+    Parameters
+    ----------
+    text : :class:`str`
+        The text that need to created as mobject.
+
+    Returns
+    -------
+    :class:`PangoText`
+        The mobject like :class:`.VGroup`.
+
+    Examples
+    --------
+    .. manim:: PangoRender
+        :quality: low
+
+        class PangoRender(Scene):
+            def construct(self):
+                morning = PangoText("வணக்கம்", font="sans-serif")
+                self.play(Write(morning))
+                self.wait(2)
+
+    .. WARNING::
+
+        Using a :class:`.Transform` on text with leading whitespace can look
+        `weird <https://github.com/3b1b/manim/issues/1067>`_. Consider using
+        :meth:`remove_invisible_chars` to resolve this issue.
+
+    """
+
+    CONFIG = {
+        # Mobject
+        "color": WHITE,
+        "height": None,
+        "width": None,
+        "fill_opacity": 1,
+        "stroke_width": 0,
+        "should_center": True,
+        "unpack_groups": True,
+        # Text
+        "font": "",
+        "gradient": None,
+        "line_spacing": -1,
+        "size": 1,
+        "slant": NORMAL,
+        "weight": NORMAL,
+        "t2c": {},
+        "t2f": {},
+        "t2g": {},
+        "t2s": {},
+        "t2w": {},
```

Comment:
I would greatly prefer typing out these abbreviations to making people look them up when they see them.

Reply:
I also wish I should. But I don't really know how to document these `config` stuff. If you can help me where to I will surely do.

Reply:
The reason I used the same name because it is from `Text`. Pretty this was done so that this doesn't become a problem to users once they switching.

Reply:
Okay, we can handle it later.